### PR TITLE
Centre checkmark in checkbox

### PR DESCRIPTION
**Pull request by agentsim**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -92,6 +92,7 @@ You can find its changes [documented below](#070---2021-01-01).
 
 ### Fixed
 
+- Centre checkmark in checkbox ([#2036] by [@agentsim])
```
Comment (jneem) [inline CHANGELOG.md:95]: You'll also need to define the links to 2036 and agentsim at the bottom of the file.

**Review by jneem — CHANGES_REQUESTED** [review]
Thanks for the PR! There's just a minor issue with the changelog (unfortunately, those links aren't automatic)

File: CHANGELOG.md (hunk 1 above)
Comment (agentsim) [reply CHANGELOG.md:95]: Thanks, fixed.

**Review by jneem — APPROVED** [review]
Thanks, and sorry for the delay!

**Pull request merged**